content/en-US/api/fontKerning.md

```diff
@@ -0,0 +1,68 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontKerning'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontKerning
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontKerning
+---
+
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontKerning](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontKerning).**
+
+## Description
+
+The `fontKerning` property of the `CanvasRenderingContext2D` interface allows 
+the developers to set whether to use of the kerning information stored in a 
+font. Kerning defines how letters are spaced. In well-kerned fonts, this 
+feature makes character spacing more uniform and pleasant to read than it
+would otherwise be. 
+
+It's illustrated in the image below. The left image turns the kerning off and 
+the right image turns the kerning on. The default value is `"auto"`.
+
+![picture](font-kerning.png)
```

Comment:
Looks like this image doesn't exist?

Reply:
weird, i attached a picture file in the same directory, file name: content/en-US/api/font-kerning.png. It works in my local directory.